src/main/webapp/src/components/Phones.js

```diff
@@ -159,6 +160,22 @@ class Phones extends React.Component {
         this.applyFilters();
     }
 
+    addToCart = (index) => {
+        let currentPhone = null;
+        this.state.phones.forEach( phone => {
+            if (phone.id === index) {
```

Comment:
is it correct? id is not the same as index

Reply:
+